private-templates-service/adaptivecards-templating-service/src/TemplateServiceClient.ts

```diff
@@ -1003,7 +1003,8 @@ export class TemplateServiceClient {
 
       let isPublished: boolean | undefined = req.query.isPublished ? req.query.isPublished === "true" || req.query.isPublished === "True" : undefined;
       let owned: boolean | undefined = req.query.owned ? req.query.owned === "true" || req.query.owned === "True" : undefined;
-      let isClient: boolean = req.body.isClient;
+      let isClient: boolean | undefined = req.query.isClient ? req.query.isClient === "true" || req.query.isClient === "True" : undefined;
+      
       if (!req.is("application/json")) {
         isClient = req.body.isClient === "true" || req.body.isClient === "True";
```

Comment:
would it not be better to have "req.query.isClient.toLowerCase( ) === "true" instead of doing everything twice?